src/LinuxTracing/TracerImpl.cpp

```diff
@@ -105,6 +105,14 @@ TracerImpl::TracerImpl(
         instrumented_function.record_arguments(), instrumented_function.record_return_value());
   }
 
+  for (const orbit_grpc_protos::FunctionToStopUnwindingAt& function_to_stop_unwinding_at :
+       capture_options.functions_to_stop_unwinding_at()) {
+    auto [unused_it, inserted] =
+        absolute_address_to_size_of_functions_to_stop_unwinding_at_.insert_or_assign(
+            function_to_stop_unwinding_at.absolute_address(), function_to_stop_unwinding_at.size());
+    ORBIT_CHECK(inserted);
```

Comment:
So this enforces that the client doesn't send duplicates. I'm always a bit torn on this because we are checking a behavior of the client (different process on a different machine...), not of the service. Maybe it's fine, but simply logging an error instead might be less drastic.

Reply:
I am pretty much in favour of the CHECK. If an absolute address is occurring twice, something is really off. We will never notice with the error log.

Also while you are right that the client is a different process on a different machine, we still develop and ship it together with the service. So I still see them as a union.